meetings/2024-01-31.md

```diff
@@ -41,28 +49,95 @@
+* RG: You looked at what changes can be made to the charter, I looked at what changes can be made to the spec.
+* PAC: I see your point, but I think it's more important to focus on the charter. I am trying to lower the importance of the spec, and show that the WG will not be stuck on one particular solution. Most of the AC reviewer thought the group is not open minded enough. 
+* eP: I am looking up at this particular paragraph:
+  > When possible, the Solid Protocol will evolve while maintaining a high degree of compatibility with existing implementations, of both servers and clients, and with features from prior versions. If incompatible changes have to be made, then they will be done by introducing a stage where both old and new protocols are supported, to allow the implementors to upgrade their systems in a managed way.
+
+* SC: Even though there was a CG for Annotations for example, the charters for Web Annotation and Social Web still propoesd their works as inputs as opposed to 'the' solution to the problem space.  These are examples, processes and people may have changed. Solid just happens to be one of the many solutions out there. Why is this locked into just Solid. I tried to differentiate between the Social Web and Solid. Sometimes we get tripped out by what Social is.
+* VB: It would be good for people to subscribe to repo and chime in, to make it faster.
+* SC: We're rewriting, are we co-writing? PAC leading? It doesn't take much to write a charter.
+* PAC: Ultimately, this is the team proposing the charter. I am trying to be as open and transparent and possible. I am happy to take the lead and submit a new version. What I want to be sure is to let this group know that I want feedback, but I want to be as responsive as possible. Submitting every PR would take too much time. Is that ok, SC?
+* SC: I just want to make the distinction. If the charter comes from the CG, then it has to be collaborative work. 
+* VB: PAC, want to respond?
+* eP: I support PAC taking full leadership. I don't think we should slow things down. Maybe before submitting having one more round of reviews.
+* PAC: Thanks eP. I am willing to hear as much feedback as I can. I will not reframe while merging things. I commit to asking the CG for a formal approval before submitting.
+* SC: Again, if it's CG's work item, I would consider that as a topic on agenda. If we leave it all the way to the end I am not sure if we'll be able to address all the point. This group was as fast as it could've in processing the issues and PRs in the past. I trust PAC's judgement, but I still think the CG should be plugged in.
+* RG: is the group willing to do a Special Actions Meeting with PAC leading to resolve this.
+* PAC: by asking this question I didn't want to seem like I am blaming the CG for being slow. Again, I welcome feedback. If we need an extensive discussion on some points, I'll try to attend these meetings.
```

Comment:
```suggestion
* SC: Again, if it's CG's work item, I would consider that as a topic on agenda. If we leave it all the way to the end, I am not sure we'll be able to address all the points. This group was as fast as it could've been in processing the issues and PRs in the past. I trust PAC's judgement, but I still think the CG should be plugged in.
* RG: Is the group willing to do a Special Actions Meeting with PAC leading to resolve this?
* PAC: I didn't want to seem like I am blaming the CG for being slow, by asking this question. Again, I welcome feedback. If we need an extensive discussion on some points, I'll try to attend these meetings.
```